xml_inscriptions/transcribed/MA.Bos.MFA.G.69.1095.xml

```diff
@@ -174,9 +174,9 @@
    <text>
       <body>
          <div type="edition">
-            <ab
-               <lb n="1"/>Μαμα Ἀπφου, Ἀπφους Εὐφρονί
-               <lb n="2" break="no"/>ου.
+            <ab>
```

Comment:
The original <lb> markup (in the previous version of the file) was correct; please revert.